Rakefile

```
@@ -52,6 +53,64 @@ task :validate_kit_versions do
   puts "Kit versions validated successfully"
 end
 
+task :validate_interactive_troubleshooter_questions => [:build] do
+  # Check that each of the next_question ids actually exists and that each
+  # question is actually used.
+
+  ROOT_QUESTION_ID = ''
+  data = YAML.load_file('resources/troubleshooter/data.json')
+  questions = data['questions']
+
+  targets = Set[]
```

```suggestion
  targets = Set.new
```

The syntax really threw me off